css/icon-button.css

```diff
@@ -0,0 +1,3 @@
+.btn-icon .btn-icon-label {
```

Comment:
Why not inlining those csss? It doesn't seem to me that you are going to reuse those across components, given the specific class names.